### PR TITLE
arrayLDEquals

### PR DESCRIPTION
**Pull request by caldwellc1**
Passes both tests. I'm going to make the right one branch off of this to try and reduce the conflicts so you might see the code again in the RDEquals.

### CODE REVIEW

File: Javatrix.java [hunk 1]
```diff
@@ -477,5 +477,31 @@ public Javatrix times(double scalar)
         }
         return actual;
     }
+    /**
+     * Return calling matrix A = A \ B.
+     *
+     * @param actualB calling matrix
```
Comment (haietza) [inline Javatrix.java:483]: actualB is not the calling matrix - it's the parameter.

File: Javatrix.java [hunk 2]
```diff
@@ -477,5 +477,31 @@ public Javatrix times(double scalar)
...
+            {
+                a = this.get(i, j);
+                b = actualB.get(i, j);
+                c = b / a;
+                this.set(i, j, c);
+            }
+        }
+        return this;
+    }
+
```
Comment (haietza) [inline Javatrix.java:505]: Remove extra line.

**Review by haietza — CHANGES_REQUESTED** [review]
Everything else looks good!

File: Javatrix.java (hunk 2 above)
Comment (caldwellc1) [reply Javatrix.java:505]: @haietza Not sure what you mean. There is no extra line.

Comment (haietza) [reply Javatrix.java:505]: @caldwellc1 Sorry about that - eyes playing tricks on me (I'm not doing a very good job of practicing energized work right now...)! Disregard!

**Review by haietza — APPROVED** [review]

**Pull request merged**